web/templates/packages/resources.tmpl

```diff
@@ -21,7 +21,7 @@
             </dd>
             <dd>
                 <span class="fa fa-fw fa-sort-numeric-desc"></span>
-                <a href="https://repology.org/project/{{.Package.Name}}" target="_blank">
+                <a href="https://repology.org/tools/project-by?repo=gentoo&name_type=srcname&target_page=project_versions&name={{.Package.Categoy}}/{{Package.Name}}" target="_blank">
```

Comment:
```suggestion
                <a href="https://repology.org/tools/project-by?repo=gentoo&name_type=srcname&target_page=project_versions&name={{.Package.Category}}/{{Package.Name}}" target="_blank">
```

After changing it, it started working on local instance. I will fix by myself and push to `develop` branch.